scripts/migrate_from_gmaven.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Replace the load statements
+find ./ -type f -name BUILD* -exec sed -i 's/gmaven_rules/rules_maven/g' {} \;
+
+# Replace the loaded gmaven_artifact symbol with maven_artifact
+find ./ -type f -name BUILD* -exec sed -i 's/"gmaven_artifact"/maven_artifact = "artifact"/g' {} \;
+
+# Replace the gmaven_artifact macro with maven_artifact
+find ./ -type f -name BUILD* -exec sed -i 's/gmaven_artifact(/maven_artifact(/g' {} \;
+
+# Remove the explicit packaging type from the coordinates
+find ./ -type f -name BUILD* -exec sed -i '/maven_artifact(/s/:aar//g; /maven_artifact(/s/:jar//g;' {} \;
+
+echo ""
+echo "Paste this into your WORKSPACE:"
+echo "-------------------------"
+echo ""
+
+cat <<-EOF
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+RULES_MAVEN_TAG = "0.1.4"
+RULES_MAVEN_SHA = ""
+
+http_archive(
+    name = "rules_maven",
+    strip_prefix = "rules_maven-%s" % RULES_MAVEN_TAG,
+    sha256 = RULES_MAVEN_SHA,
+    url = "https://github.com/jin/rules_maven/archive/%s.zip" % RULES_MAVEN_TAG,
```

Comment:
I don't think this should be published with this URL yet but rather with its final resting place

Reply:
sg, I can put this on hold.